### ISSUE TITLE
Cannot make changes to image details with the pencil icon in the classic block unless the same edit is applied twice

### ISSUE THREAD

**Issue by turtlepod** [WordPress/gutenberg#11433]
**Describe the bug**
Image In Classic Editor Block is Broken (Gutenberg Version 4.1.1/WordPress 4.9.8)

- Inserting image always adding it to first in editor
- editing not working.

**To Reproduce**
1. Create classic editor block
2. Add image in content and edit

Video : https://drive.google.com/file/d/1AKG4lm8Otq6uBGJhPgeIbF1QvyAUMARU/view?usp=sharing

**Expected behavior**
Not broken

**Screenshots**
Video: https://drive.google.com/file/d/1AKG4lm8Otq6uBGJhPgeIbF1QvyAUMARU/view?usp=sharing

**Desktop (please complete the following information):**
 - OS: win10
 - Browser chrome 70

**Comment by jesscga** [comment]
I was able to recreate this issue.

I added a thumbnail (150x150) image to a Classic block.

After adding, I was able to change alignment, but not the image size. I clicked the edit button, changed the image size to 800x800, and went back to the block. The image was still a thumbnail and the HTML view showed the original 150x150 size.

Screencast: http://cld.wthms.co/YRTMA2

OS: macOC 10.13.6 High Sierra
Browser: Chrome Version 69.0.3497.100
WordPress 4.9.8
Gutenberg 4.1.1

**Comment by La-Geek** [comment]
The first issue (image that should be below, but is on top after adding) was described in [#10509](https://github.com/WordPress/gutenberg/issues/10509)
but the ticket was hastily closed (why one day later, after the issue has been confirmed here a second time?)

The second issue (resizing image): Confirmed, does work only after the second time, you have to resize a second time, then it works.

**Comment by arunsathiya** [comment]
I came here to report the behaviour with image addition at the top, but glad that it's been tracked here https://github.com/WordPress/gutenberg/issues/10509.

>but not the image size. I clicked the edit button, changed the image size to 800x800, and went back to the block. The image was still a thumbnail and the HTML view showed the original 150x150 size.

I was able to reproduce this as well. My understanding is that, trying to `Preview` the image size changes and then publishing/updating the post/page reflects the changes. Trying to publish/update without previewing doesn't reflect the changes - or in this case, one will need to update the image size twice to see the changes.

**Comment by mtias** [comment]
Closing in favor of #10509.

**Comment by turtlepod** [comment]
@mtias There's 2 bug in this issue:
1. inserting image
2. editing the image (change the image size, etc)

you close this for #10509 
Have you create a new issue for # 2?

**Comment by turtlepod** [comment]
Nevermind, I create a new issue. Feel free to close if duplicate.
Thanks.

**Comment by designsimply** [comment]
Re-tested and confirmed using WordPress 4.9.8 and Gutenberg`53e4c40bd` (latest master) with Firefox 63.0.1 on macOS 10.13.6 that I cannot make change to an image in the classic block using the pencil icon in the image toolbar unless I make the same adjustment twice. ([1m39s](https://cloudup.com/cebk8E0Dae7))

![untitled](https://user-images.githubusercontent.com/1119271/48860951-94549200-ed7f-11e8-8132-8c89fe37d268.png)

**Comment by kristarella** [comment]
I was able to replicate with Gutenberg 4.6.1.

I found that changing the image size of an image added in another editor was inconsistent, until the post was saved in Gutenberg, and then the image size was able to be changed consistently.

Here is my test recording (motorbikes were after post save, see second test in recording with pug to see inconsistency):
![gute-classic-edit-image-atomic](https://user-images.githubusercontent.com/2031017/49324845-698de900-f58a-11e8-92a6-6feea39d8976.gif)

**Comment by JenniV** [comment]
I am having this issue as well. I need to change the link twice for it to appear correctly and it's SUPER ANNYOING.   Also, every time I change a link the page moves all the way back up to the top so I have to scroll all the way down to continue working. No biggie on a small post, but a long one it's taking me forever. How can I fix this or just turn this new update off.

**Comment by azaozz** [comment]
Opened https://core.trac.wordpress.org/ticket/46085 to track this as it's a bug in the `wpeditimage` WP plugin for TinyMCE. Probably going to be fixed in WP 5.1.

**Comment by azaozz** [comment]
Lets move any further discussion/bug reports there :)
